CMakeLists.txt

```diff
@@ -2737,13 +2737,13 @@ string(REPLACE "\n\n" "\n.\n" CPACK_DEBIAN_PACKAGE_DESCRIPTION_MERGED "${CPACK_D
 string(REPLACE "\n" "\n " CPACK_DEBIAN_PACKAGE_DESCRIPTION_MERGED "${CPACK_DEBIAN_PACKAGE_DESCRIPTION_MERGED}")
 
 # The upstream version must not contain hyphen
-string(REPLACE "-" "~" CPACK_DEBIAN_UPSTREAM_VERSION "${MIXXX_VERSION}")
-set(CPACK_DEBIAN_DEBIAN_VERSION 0ubuntu1)
-set(CPACK_DEBIAN_DEBIAN_VERSION_EXTRA "${GIT_BRANCH}~git${GIT_COMMIT_COUNT}")
-string(REPLACE "_" "" CPACK_DEBIAN_DEBIAN_VERSION_EXTRA "${CPACK_DEBIAN_DEBIAN_VERSION_EXTRA}")
-set(CPACK_DEBIAN_PACKAGE_VERSION "${CPACK_DEBIAN_UPSTREAM_VERSION}-ppa1~${GIT_BRANCH}~git${GIT_COMMIT_COUNT}~bionic")
+string(REPLACE "-" "~" CPACK_DEBIAN_PACKAGE_VERSION "${PACKAGE_VERSION}")
+set(CPACK_DEBIAN_PACKAGE_RELEASE mixxx1)
```

Comment:
Typo? If not, please add a comment why it's `mixxx1`.

Reply:
No, but we may change it because that a bit unusual. 

This is the package version. Ubuntu uses 0ubuntu1 where the first number is the debian version and the last number the patched version. Debian version packages dfsg1. The lint tools is complaining that we must not use one of these ending because we have not the debian/ubuntu email address. 

Alternatives 
ppa1
build1 

Other option?

Reply:
I don't care, we can keep using `mixxx1`, just add that explanation as a comment so other people don't think it's a typo when reading the code.

Reply:
I have picked now just 1 for the least confusing version.